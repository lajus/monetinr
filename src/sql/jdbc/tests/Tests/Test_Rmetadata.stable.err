stderr of test 'Test_Rmetadata` in directory 'sql/jdbc/tests` itself:


# 22:09:00 >  
# 22:09:00 >  Mtimeout -timeout 240 Mserver "--config=/net/pictor.ins.cwi.nl/export/scratch1/fabian/monet-stable/program/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/net/pictor.ins.cwi.nl/export/scratch1/fabian/monet-stable/program/lib/MonetDB" --set "gdk_dbfarm=/net/pictor.ins.cwi.nl/export/scratch1/fabian/monet-stable/program/var/MonetDB/dbfarm" --set "sql_logdir=/net/pictor.ins.cwi.nl/export/scratch1/fabian/monet-stable/program/var/MonetDB/log" --set mapi_port=48319 --set sql_port=55250 --set monet_prompt= --trace "--dbname=mTests_src_jdbc_tests" "/net/pictor.ins.cwi.nl/export/scratch1/fabian/monet-stable/sql/build/src/backends/monet4/sql_server.mil" ; echo ; echo Over..
# 22:09:00 >  


# 22:09:01 >  
# 22:09:01 >  Mtimeout -timeout 120 ./Test_Rmetadata.SQL Test_Rmetadata 
# 22:09:01 >  


# 22:09:01 >  
# 22:09:01 >  java Test_Rmetadata jdbc:monetdb://pictor:55250/database?user=monetdb&password=monetdb
# 22:09:01 >  


# 22:09:01 >  
# 22:09:01 >  Done.
# 22:09:01 >  

