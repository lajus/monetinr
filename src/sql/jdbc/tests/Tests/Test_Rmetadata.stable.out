stdout of test 'Test_Rmetadata` in directory 'sql/jdbc/tests` itself:


# 22:09:00 >  
# 22:09:00 >  Mtimeout -timeout 240 Mserver "--config=/net/pictor.ins.cwi.nl/export/scratch1/fabian/monet-stable/program/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/net/pictor.ins.cwi.nl/export/scratch1/fabian/monet-stable/program/lib/MonetDB" --set "gdk_dbfarm=/net/pictor.ins.cwi.nl/export/scratch1/fabian/monet-stable/program/var/MonetDB/dbfarm" --set "sql_logdir=/net/pictor.ins.cwi.nl/export/scratch1/fabian/monet-stable/program/var/MonetDB/log" --set mapi_port=48319 --set sql_port=55250 --set monet_prompt= --trace "--dbname=mTests_src_jdbc_tests" "/net/pictor.ins.cwi.nl/export/scratch1/fabian/monet-stable/sql/build/src/backends/monet4/sql_server.mil" ; echo ; echo Over..
# 22:09:00 >  

# Monet Database Server V4.4.3
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.


Ready.


# 22:17:15 >  
# 22:17:15 >  Mtimeout -timeout 120 ./Test_Rmetadata.SQL Test_Rmetadata 
# 22:17:15 >  


# 22:17:15 >  
# 22:17:15 >  java Test_Rmetadata jdbc:monetdb://pictor:59613/database?user=monetdb&password=monetdb
# 22:17:15 >  

0. false	false
0. 4 columns:	5
1.	mTests_sql_jdbc_tests
	classname     java.lang.Integer
	displaysize   1
	label         myint
	name          myint
	type          4
	typename      int
	precision     32
	scale         0
	schemaname    sys
	tablename     table_test_rmetadata
	autoincrement false
	casesensitive false
	currency      false
	defwritable   false
	nullable      1
	readonly      true
	searchable    true
	signed        true
	writable      false
2.	mTests_sql_jdbc_tests
	classname     java.lang.Double
	displaysize   24
	label         mydouble
	name          mydouble
	type          8
	typename      double
	precision     53
	scale         0
	schemaname    sys
	tablename     table_test_rmetadata
	autoincrement false
	casesensitive false
	currency      false
	defwritable   false
	nullable      1
	readonly      true
	searchable    true
	signed        true
	writable      false
3.	mTests_sql_jdbc_tests
	classname     java.lang.Boolean
	displaysize   5
	label         mybool
	name          mybool
	type          16
	typename      boolean
	precision     1
	scale         0
	schemaname    sys
	tablename     table_test_rmetadata
	autoincrement false
	casesensitive false
	currency      false
	defwritable   false
	nullable      1
	readonly      true
	searchable    true
	signed        false
	writable      false
4.	mTests_sql_jdbc_tests
	classname     java.lang.String
	displaysize   8
	label         myvarchar
	name          myvarchar
	type          12
	typename      varchar
	precision     15
	scale         0
	schemaname    sys
	tablename     table_test_rmetadata
	autoincrement false
	casesensitive false
	currency      false
	defwritable   false
	nullable      1
	readonly      true
	searchable    true
	signed        false
	writable      false
5.	mTests_sql_jdbc_tests
	classname     java.sql.Clob
	displaysize   11
	label         myclob
	name          myclob
	type          2005
	typename      clob
	precision     0
	scale         0
	schemaname    sys
	tablename     table_test_rmetadata
	autoincrement false
	casesensitive false
	currency      false
	defwritable   false
	nullable      1
	readonly      true
	searchable    true
	signed        false
	writable      false
5.	(null)
5.	(null)
5.	(null)
5.	(null)
5.	(null)
6.	java.lang.Integer is an instance of java.lang.Integer
6.	java.lang.Double is an instance of java.lang.Double
6.	java.lang.Boolean is an instance of java.lang.Boolean
6.	java.lang.String is an instance of java.lang.String
6.	nl.cwi.monetdb.jdbc.MonetClob is an instance of java.sql.Clob

# 09:09:21 >  
# 09:09:21 >  Done.
# 09:09:21 >  

